common/app/views/support/cleaner/AmpAdCleaner.scala

```diff
@@ -88,7 +88,7 @@ case class AmpAdCleaner(edition: Edition, uri: String, article: Article) extends
 
   def adAfter(element: Element) = {
     val ampAd = <div class="amp-ad-container">
-          <amp-ad width="300" height="250" type="doubleclick" data-loading-strategy="prefer-viewability-over-views"
+          <amp-ad width="300" height="250" type="doubleclick" data-use-beta-a4a-implementation="true" data-loading-strategy="prefer-viewability-over-views"
```

Comment:
Should this be here in dev? If it only serves a purpose in PROD maybe it needs an if statement around it? However, it might not hurt to just have it in always :). Also, since it throws as error, maybe it deserves a comment?